### PR TITLE
Don't pretent we can work without pthread

### PR DESCRIPTION
**Pull request by yschaeff**

### CODE REVIEW

**Comment by yschaeff** [comment]
#build

**Comment by yschaeff** [comment]
Merge at will

File: enforcer/src/scheduler/schedule.c [hunk 1]
```diff
@@ -62,9 +62,12 @@ static task_t* get_first_task(schedule_type *schedule);
 static void*
 alarm_handler(sig_atomic_t sig)
 {
+    pthread_t tid = pthread_self();
     switch (sig) {
         case SIGALRM:
+        
             ods_log_debug("[%s] SIGALRM received", schedule_str);
+            printf("[%s] SIGALRM received: %d\n", schedule_str, tid);
```
Comment (halderen) [inline enforcer/src/scheduler/schedule.c]: Debug output to be removed.

**Comment by halderen** [comment]
Please note that my pull requests will need to re-introduce some of the methods.

**Pull request merged**